ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -61,7 +61,9 @@
 
         setModel(model) {
             super.setModel(model);
-            this._model.value = this.widget.value;
+            if (this.widget.value !== '') {
+                this._model.value = this.widget.value;
+            }
```

Comment:
If an end user empties the value then also this thing will happen. I think we should fix that in the core.

Reply:
yes, I agree. however, no harm in having it here till we update af-core. :)